src/containers/FriendsContainer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import styled from 'styled-components' 
+import { useNavigate } from 'react-router-dom'
+import {v4 as uuidv4} from "uuid";
+
+import { useAppDispatch, useAppSelector } from '../hooks'
+import { addChat, setActiveChat, setActiveClient } from '../state/chatSlice'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: flex-start;
+  flex-wrap: wrap;
+  padding: 1rem;
+  overflow: scroll;
+  height: 60vh;
+  &::-webkit-scrollbar {
+    display:none;
+  }
+`
+
+const ProfileDiv = styled.div`
+  width: 8rem;
+  height: 10rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+  background-color: rgba(255, 255, 255, 0.5);
+  border-radius: 2rem;
+  & p{
+    font-size: 0.8rem;
+  }
+  &:hover{
+    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.1);
+  }
+`
+
+const ProfileName = styled.p`
+  border-radius: 3rem;
+  background-color: rgba(255, 255, 255, 1);
+  color: rgba(150, 150, 150, 1);
+  margin: 1rem 0 1rem 0;
+  padding: 0.2rem 1rem 0.2rem 1rem;
```

Comment:
```suggestion
  margin: 1rem 0;
  padding: 0.2rem 1rem;
```

좌우나 상하에 중복된 ```margin```이나 ```padding```을 사용하는 경우 이렇게 짧게 적어줘도 좋을 것 같아요!

Reply:
맞아요, 이렇게도 가능했었죠, 다음부터는 참조해서 적어보도록 할게요!